incuna-sass/mixins/_svg-background.sass

```diff
@@ -1,12 +0,0 @@
-@mixin svg-background($svg, $inline: false, $width: '', $height: '', $dimensions: false)
```

Comment:
no, this is being replace with those two mixins I made before we started looking at grunticon, they're not just for icons :)